### ISSUE TITLE
Scroll not working after update

### ISSUE THREAD

**Issue by naitikAIITS** [a85/POSTMan-Chrome-Extension#12824]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

after adding new request 
go to Authentication tab
then select Auth Type : here the scroll bar vont allow to scroll up or down  

### Steps To Reproduce

1. open new request
2. then go to authentication tab
3. auth type
4. scroll not working 

### Operating System

Windows

### Postman Version

11.0.5

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by bhargavkaranam** [comment]
@naitikAIITS Could you post a screenshot/screen recording of the behaviour you're seeing?

**Comment by naitikAIITS** [comment]
![Screenshot (4)](https://github.com/postmanlabs/postman-app-support/assets/159764477/414fc3f3-d89e-4312-ae4d-41b29d494030)

**auth type select option**:
here im using it in small tab so the options should have to scroll as per earlier, but it can be access using arrow keys and in full screen

**Comment by sandeep-singh-pstmn** [comment]
@naitikAIITS, @DBitao Thanks for reporting this. We will investigate and in the meantime, please try opening Postman in fullscreen mode to unblock.

**Comment by DanilVanDovgal** [comment]
Hello!
I've just updated to version 11.0.6 and have the same problem, I can't scroll the list.

**Comment by Postmanaut-Adrian** [comment]
Hello Everyone, 

While our teams work on investigating and resolving this issue, we found an interesting workaround. 

If you import a curl command with an authorization set, then the menu will populate as expected

- Example: 

```
 curl --location 'https://www.postman-echo.com/get' \
--header 'Authorization: Bearer adfsdfs'
```

We hope this helps mitigate the issue while we work on a more permanent and elegant fix.

**Comment by bhargavkaranam** [comment]
This should now be fixed as part of v11.0.7. Please update your app. Feel free to re-open the issue if you're still facing the same problem.
